src/main/surfrdMod.F90

```diff
@@ -877,14 +887,14 @@ subroutine surfrd_veg_all(begg, endg, ncid, ns, actual_numcft)
     !                 variable to 0 where is_pft_known_to_model = .false.?
     call collapse_crop_var(fert_cft(begg:endg,:), cft_size, begg, endg)
 
-    ! Call collapse_nat_pfts: enhance ctsm performance with fewer active pfts
+    ! Call collapse_to_dominant: enhance ctsm performance with fewer active pfts
     ! Collapsing to the top N dominant pfts (n_dom_pfts set in namelist).
     ! - Bare ground could be up to 1 patch before collapsing.
     ! - Pfts could be up to 14 before collapsing if create_crop_landunit = .T.
     ! - Pfts could be up to 16 before collapsing if create_crop_landunit = .F.
-    ! The call to collapse_nat_pfts also appears in subroutine dynpft_interp
-    call collapse_nat_pfts(wt_nat_patch(begg:endg,:), natpft_size, &
-                           begg, endg, n_dom_pfts)
+    ! The call does not appear in subroutine dynpft_interp: SHOULD IT?
```

Comment:
@slevisconsulting do we know the answer to this question? Do we need to test n_dom_landunits for transient cases? Since, you've done the testing we should bring this to master. But, we should make sure this question is answered soon.